src/System.Management.Automation/engine/regex.cs

```diff
@@ -148,15 +148,8 @@ private void Init()
                 return;
             }
 
-            if (Pattern.Length == 1 && Pattern[0] == '*')
+            StringComparison GetStringComparison()
```

Comment:
Please move this local function to the beginning of this method.

Reply:
Fixed.